lib/puppet/type/yumrepo.rb

```diff
@@ -321,7 +321,7 @@
 
     munge do |value|
       return '' if Facter.value(:operatingsystemmajrelease).to_i >= 8 && value.to_s == '_none_'
-      value.to_s
+      super(value)
```

Comment:
The [docs](https://puppet.com/docs/puppet/latest/custom_types.html) suggest `super` instead of `super(value)`.  They need updating though?
Otherwise you get
```
Munging failed for value "http://proxy.example.com:3128" in class proxy: implicit argument passing of super from method defined by define_method() is not supported. Specify all arguments explicitly.
```

Reply:
Thanks for the heads-up, We'll create a docs ticket for this.

Reply:
I already left feedback on the page, so I guess a ticket should magic into existence anyway?

Reply:
Docs fixed! Thanks. 😄